lib/plug/adapters/cowboy2.ex

```diff
@@ -268,6 +301,21 @@ defmodule Plug.Adapters.Cowboy2 do
     end
   end
 
+  defp option_deprecation_warning(option, expected_option \\ nil)
```

Comment:
I think we can skip the default arg here and call `option_deprecation_warning/1`  that calls `option_deprecation_warning/2`.